src/main/java/org/jabref/gui/preferences/AppearancePrefsTab.java

```diff
@@ -53,9 +53,10 @@ public AppearancePrefsTab(DialogService dialogService, JabRefPreferences prefs)
         darkTheme = new RadioButton("Dark theme");
         darkTheme.setToggleGroup(themeGroup);
 
-        if (prefs.get(JabRefPreferences.FX_THEME).equals(BASE_CSS))
+        String cssFileName = prefs.get(JabRefPreferences.FX_THEME);
+        if (BASE_CSS.equals(cssFileName))
```

Comment:
The light theme should also be selected by default:

```suggestion
        if (StringUtil.isBlank(cssFileName) || BASE_CSS.equals(cssFileName))
```

Moreover, please add brackets {} surrounding the if and else if parts.